Android/Nothing/build.gradle

```diff
@@ -9,7 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:4.2.0-alpha13'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.3.2'
+        classpath 'com.google.gms:google-services:4.3.4'
```

Comment:
Can we move all versions inside ext block?

Reply:
Better implementation for dependencies will be in a different PR

Reply:
Maybe in a buildSrc. Yeah! I know the exit!